### PR TITLE
Data quality and data governance quality introduction

### PR DESCRIPTION
**Pull request by thijs-franck**
Added an introduction for the data quality and data governance quality sections

### CODE REVIEW

File: DOc/docs/source/Options/user-story-5.rst [hunk 1]
```diff
@@ -11,21 +11,21 @@ Keep track of your data quality
...
-like Aldus PageMaker including versions of Lorem Ipsum.
-
-Why do we use it?
-It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. 
-The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', 
-making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 
-'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose 
-(injected humour and the like).
- 
+When first starting to set up a data management organisation, there are a lot of different aspects to consider.
```
Comment (AmericoChiclana) [inline DOc/docs/source/Options/user-story-5.rst]: When setting up a data management organisation for the first time.

File: DOc/docs/source/Options/user-story-5.rst [hunk 2]
```diff
@@ -11,21 +11,21 @@ Keep track of your data quality
...
-
-Why do we use it?
-It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. 
-The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', 
-making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 
-'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose 
-(injected humour and the like).
- 
+When first starting to set up a data management organisation, there are a lot of different aspects to consider. 
+The data governance organisation needs to be set up.
```
Comment (AmericoChiclana) [inline DOc/docs/source/Options/user-story-5.rst:14]: delete this sentence is redundant

File: DOc/docs/source/Options/user-story-5.rst [hunk 3]
```diff
@@ -11,21 +11,21 @@ Keep track of your data quality
...
+Business and technical data governance models need to be set up.
+And data quality needs to be monitored.
+
+Keeping an overview of the organisation's progress in all these activities can be a challenge.
```
Comment (AmericoChiclana) [inline DOc/docs/source/Options/user-story-5.rst:18]: overview of the model progress can be challenging

File: DOc/docs/source/Options/user-story-5.rst [hunk 4]
```diff
@@ -11,21 +11,21 @@ Keep track of your data quality
...
+The Aurelius Atlas tool lets key stakeholders monitor the progress of the implementation, as well as the overall data governance quality.
```
Comment (AmericoChiclana) [inline DOc/docs/source/Options/user-story-5.rst]: the entire progress of the implementation, as well of the overall data.....

File: DOc/docs/source/Options/user-story-5.rst [hunk 5]
```diff
@@ -11,21 +11,21 @@ Keep track of your data quality
...
+This allows the organisation to monitor its progress in applying data management.
+It also allows for better decision making on where to focus for the next improvement.
```
Comment (AmericoChiclana) [inline DOc/docs/source/Options/user-story-5.rst:20]: It allows you to make better decisions and clarifies what to focus on for a next  improvement

File: DOc/docs/source/Options/user-story-5.rst [hunk 6]
```diff
@@ -11,21 +11,21 @@ Keep track of your data quality
...
+
+Keeping an overview of the organisation's progress in all these activities can be a challenge.
+The Aurelius Atlas tool lets key stakeholders monitor the progress of the implementation, as well as the overall data governance quality.
+This allows the organisation to monitor its progress in applying data management.
+It also allows for better decision making on where to focus for the next improvement. 
+
+Every entity in Aurelius Atlas is checked against various data governance quality rules.
+The result of this check is available at the bottom of every details page, as well as an indicator in the search results.
+An entity can either be compliant with a rule, or not.
+All compliant and non-compliant rules are counted, and the ratio determines an overall data governance quality score.
```
Comment (AmericoChiclana) [inline DOc/docs/source/Options/user-story-5.rst]: delete the coma after counted

File: DOc/docs/source/Options/user-story-4.rst [hunk 7]
```diff
@@ -10,22 +10,43 @@ Find data fast and observed quality metrics
...
-Why do we use it?
-It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. 
-The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', 
-making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 
-'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose 
-(injected humour and the like).
 
+Next to data governance, data quality is another core pillar of data management. 
+Good data quality helps make data more useful and usable, resulting in better business outcomes.
+Meanwhile, bad data quality limits the value of data, and results in worse business outcomes.
```
Comment (AmericoChiclana) [inline DOc/docs/source/Options/user-story-4.rst]: delete the coma after data

File: DOc/docs/source/Options/user-story-4.rst [hunk 8]
```diff
@@ -10,22 +10,43 @@ Find data fast and observed quality metrics
...
+
+For every dimension, one or more data quality rules can be defined.
+For every rule, a percentage score is assigned. 
+A score of 100% indicates that the dataset is perfectly compliant with that particular rule.
+
+A lower data quality score for a particular rule indicates that there are data quality issues.
+The impact and consequences of these issues depends on the characteristics of the dataset.
+
+Typically, manually entered data tends to be of lesser quality than automatically generated data.
+Improving the data quality of manually entered data tends to involve the education of people and process improvements.
```
Comment (AmericoChiclana) [inline DOc/docs/source/Options/user-story-4.rst]: put a coma after "entered data"

File: DOc/docs/source/Options/user-story-4.rst [hunk 9]
```diff
@@ -10,22 +10,43 @@ Find data fast and observed quality metrics
...
+Meanwhile, lower data quality scores for automatically generated data may indicate a bug in the system.
+
+Data quality should be measured at every system where the data is stored, since data transformations can cause data quality to change.
+If there are data quality issues, these should be solved at the point where the data quality issues are introduced.
+This is typically at the place where the master data is stored.
+
+Aurelius Atlas allows for the discovery and insight of your organisation's data quality across data sources.
```
Comment (AmericoChiclana) [inline DOc/docs/source/Options/user-story-4.rst]: just organisation without the apostrophe

File: DOc/docs/source/Options/user-story-4.rst [hunk 10]
```diff
@@ -10,22 +10,43 @@ Find data fast and observed quality metrics
...
+This allows for better business outcomes for users of data.
+The tool also allows the people responsible for data quality to do root cause analysis of data quality issues.
```
Comment (AmericoChiclana) [inline DOc/docs/source/Options/user-story-4.rst]: put a coma after "data quality, to do a root"

**Review by AmericoChiclana — CHANGES_REQUESTED** [review]

File: DOc/docs/source/Options/user-story-4.rst [hunk 11]
```diff
@@ -10,22 +10,43 @@ Find data fast and observed quality metrics
...
+| Dimension    | Description                                                                                                             |
+|--------------|-------------------------------------------------------------------------------------------------------------------------|
+| Accuracy     | Whether or not the data contains errors. For example, a person's name should always be correct.                         |
+| Completeness | Whether or not each row of the data has a value. For example, a person should always have a name.                       |
+| Timeliness   | Whether or not the data is up to date. For example, the data should only include people currently employed.             |
+| Uniqueness   | Whether or not values that need to be unique, actually are unique. For example, every employee should have a unique ID. |
+| Validity     | Whether or not the data fits with a predefined format. For example, a person's initials should always be capitalized.   |
+
+For every dimension, one or more data quality rules can be defined.
+For every rule, a percentage score is assigned.
```
Comment (wombach) [inline DOc/docs/source/Options/user-story-4.rst]: For every rule, a percentage score is calculated by dividing the number of compliant entities over the number of all entities,

File: DOc/docs/source/Options/user-story-4.rst [hunk 12]
```diff
@@ -10,22 +10,43 @@ Find data fast and observed quality metrics
...
+
+Data quality should be measured at every system where the data is stored, since data transformations can cause data quality to change.
+If there are data quality issues, these should be solved at the point where the data quality issues are introduced.
+This is typically at the place where the master data is stored.
+
+Aurelius Atlas allows for the discovery and insight of an organisation's data quality across data sources.
+This allows for better business outcomes for users of data.
+The tool also allows the people responsible for data quality, to do root cause analysis of data quality issues.
+
+The following example demonstrates how data of good quality is discovered by a data scientist.
```
Comment (wombach) [inline DOc/docs/source/Options/user-story-4.rst:51]: This is a detail/lengthy introduction of what data governance is and how we look at it. The first two sentences state what is the problem, the subsequent sentence states why it is a problem. The rest addresses how we solve the problem.
Can you add some more details on why it is a problem. What are the consequences if data quality is bad?
AT the end I am missing a description of a short scenario: Something like a data engineer has found the dataset NL1-HR-001 <add link> and wants to understand the data quality of the dataset.

File: DOc/docs/source/Options/user-story-5.rst (hunk 2 above)
Comment (wombach) [reply DOc/docs/source/Options/user-story-5.rst:14]: > delete this sentence is redundant

I disagree: data management and data governance organization are two fundamental different things! Please keep the sentence

File: DOc/docs/source/Options/user-story-5.rst [hunk 13]
```diff
@@ -11,22 +11,22 @@ Keep track of your data quality
...
+The Aurelius Atlas tool lets key stakeholders monitor the entire progress of the implementation, and the overall data governance quality.
+It also allows for better decision making on where to focus for the next improvement. 
+
+Every entity in Aurelius Atlas is checked against various data governance quality rules.
+The result of this check is available at the bottom of every details page, as well as an indicator in the search results.
+An entity can either be compliant with a rule, or not.
+All compliant and non-compliant rules are counted. The ratio determines an overall data governance quality score.
+This score is shown at the top of every details page.
+
+The following example explains how to review data governance quality from the perspective of a business data steward.
```
Comment (wombach) [inline DOc/docs/source/Options/user-story-5.rst:32]: AT the end I am missing a description of a short scenario: Something like a data scientist is looking for some trustworthy data for a specific topic. Already during the selection of relevant dataset he wants to have an indication on the data quality of the dataset. This should set the scene for the walk through example.

**Review by wombach — CHANGES_REQUESTED** [review]

**Review by wombach — APPROVED** [review]

File: DOc/docs/source/Options/user-story-4.rst [hunk 14]
```diff
@@ -10,22 +10,47 @@ Find data fast and observed quality metrics
...
-Why do we use it?
-It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. 
-The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', 
-making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 
-'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose 
-(injected humour and the like).
-
 
+Next to data governance, data quality is another core pillar of data management. 
+Good data quality helps make data more useful and usable, resulting in better business outcomes.
```
Comment (gabypascha) [inline DOc/docs/source/Options/user-story-4.rst:11]: and reliable

File: DOc/docs/source/Options/user-story-4.rst [hunk 15]
```diff
@@ -10,22 +10,47 @@ Find data fast and observed quality metrics
...
+Meanwhile, bad data quality limits the value of data and results in worse business outcomes. 
+Poor data quality in a source system can, for example, lead to misleading indicators on management dashboards. 
+This can lead to ineffective decision making and a waste of time and resources.
```
Comment (gabypascha) [inline DOc/docs/source/Options/user-story-4.rst:14]: change the first and to ,

**Pull request merged**